lib/common/logging.c

```diff
@@ -609,6 +609,20 @@ crm_log_filter_source(int source, const char *trace_files, const char *trace_fns
     }
 }
 
+#ifndef HAVE_STRCHRNUL
+/* strchrnul() is a GNU extension. If not present, use our own definition.
```

Comment:
Doesn't `_GNU_SOURCE` cover `strchrnul()`?

From `strchr(3)` man page:
```
       #define _GNU_SOURCE         /* See feature_test_macros(7) */
       #include <string.h>

       char *strchrnul(const char *s, int c);
```

Reply:
We don't require glibc (*BSD being the obvious exception)